amethyst_rendy/src/visibility.rs

```diff
@@ -137,7 +137,7 @@ impl System for VisibilitySortingSystem {
                             entity_query
                                 .iter(world)
                                 .map(|(entity, transform, transparent, sphere)| {
-                                    let pos = sphere.clone().map_or(origin, |s| s.center);
+                                    let pos = sphere.map_or(origin, |s| s.center);
```

Comment:
Is the clone here required?

Reply:
I remove it since clippy said it was unnecessary. You can see the green part where i remove the clone.

Reply:
oh.... :D 
I'm not sure what I was thinking, thanks for pointing it out x)